### ISSUE TITLE
Server should announce on same protocol it listens on

### ISSUE THREAD

**Issue by ShadowNinja** [minetest/minetest#3609]
Apparently servers listening on IPv4 sometimes announce over IPv6.  This causes issues with the server list because it can't verify that the announcement is coming from the real server.
The workaround is to disable the authenticity check on the master server, but this allows announcing other servers.
Related: minetest/master-server#8

**Comment by est31** [comment]
So when it listens both on ipv6 and ipv4, it should announce twice?

**Comment by est31** [comment]
At least when #2604 gets merged this is an issue.

**Comment by ShadowNinja** [comment]
@est31: No, only once.  You should set the address to a domain containing both a and aaaa records for your server.  As long as one of them matches your announcing address it should work.

**Comment by est31** [comment]
So force to use DNS? This is bad IMO.

**Comment by ShadowNinja** [comment]
@est31: If you want multi-protocol support yes.  What's a good alternative?
